src/lib/dnssd/Resolver.h

```diff
@@ -36,6 +36,12 @@ namespace Dnssd {
 
 struct ResolvedNodeData
 {
+    enum class CacheBypass
```

Comment:
IMHO it would make more sense to put this in `Resolver` class as it controls the behavior of `Resolver`, not `ResolvedNodeData`.